package.json

```diff
@@ -30,14 +30,17 @@
     }
   ],
   "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
   "types": "dist/cjs/index.d.ts",
   "files": [
     "dist",
     "LICENSE",
     "README.md"
   ],
   "scripts": {
-    "build": "tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json",
+    "build": "npm run build:cjs && npm run build:esm",
+    "build:cjs": "tsc -p tsconfig.cjs.json && echo '{ \"type\": \"commonjs\" }' > './dist/cjs/package.json'",
+    "build:esm": "tsc -p tsconfig.esm.json && echo '{ \"type\": \"module\" }' > './dist/esm/package.json'",
```

Comment:
I think that the package.json file under the cjs/esm directory probably won't be read, but is there a link  of this config information?

Reply:
It would be, this is a commonplace practice.

https://nodejs.org/api/packages.html#type

```
The "type" field defines the module format that Node.js uses for all .js files that have that package.json file as their nearest parent.
```

`have that package.json file as their nearest parent`

Reply:
Generally, placing a `{ "type": "commonjs" }` or `{ "type": "module" }` alongside the generated bundle files is an alternative to declaring a package strictly ESM by placing `"type": "module"` in the root `package.json`

Reply:
Thanks for the information! wow, I wasn’t aware of this specification.